input/pagecontent/StructureDefinition-au-erequesting-path-request-intro.md

```diff
@@ -0,0 +1,8 @@
+### Usage scenarios
+
+The following are supported usage scenarios for this profile:
```

Comment:
The usage scenario wording reads like AU Core. Remove for now, this should be added after the interactions and obligations meetings.

Reply:
Removed, although should've probably removed the heading as well rather than leaving as a placeholder.

Reply:
Remove it entirely, not just placeholder.

Reply:
Done